content/docs/run-api-server/running.mdx

```diff
@@ -69,9 +70,9 @@ horizon3> horizon db reingest range 20001 30000
 This allows reingestion to be split up and done in parallel by multiple Horizon processes.
 
 ### Managing Storage for Historical Data
-Over time, the recorded network history will grow unbounded, increasing storage used by the database. Horizon needs sufficient disk space to expand the data ingested from Stellar Core. Unless you need to maintain a [history archive](../run-core-node/publishing-history-archives.mdx), you may configure Horizon to only retain a certain number of ledgers in the database. This is done using the `--history-retention-count` flag or the `HISTORY_RETENTION_COUNT` environment variable. Set the value to the number of recent ledgers you wish to keep around, and every hour the Horizon subsystem will reap expired data. Alternatively, you may execute the command `horizon db reap` to force a collection.
+Over time, the recorded network history will grow unbounded, increasing storage used by the database. Horizon needs sufficient disk space to expand the data ingested from Stellar Core. Unless you need to maintain a [history archive](../run-core-node/publishing-history-archives.mdx), you may configure Horizon to only retain a certain number of ledgers in the database. 
```

Comment:
How do I estimate how much storage I need for a server?

Reply:
Might be worth suggesting 5s as a rule-of-thumb multiplier to get a sense of how far back in time your storage settings will go.